requirements/updater.sh

```diff
@@ -19,74 +17,36 @@ _cleanup() {
   test "${KEEP_TMP:-0}" = 1 || rm -rf "${_tmp}"
 }
 
-install_deps() {
-  pip install pip --upgrade
-  pip install "pip-tools==5.4.0"  # see https://github.com/jazzband/pip-tools/pull/1237
-}
-
-generate_requirements_v3() {
-  venv="./venv3"
-  python3 -m venv "${venv}"
+generate_requirements() {
+  venv="`pwd`/venv"
+  echo $venv
+  /usr/bin/python3 -m venv "${venv}"
```

Comment:
`python` was defaulting to `awx-python` in the path, which was mixing all this up with the awx env in the image (gross).  This is more explicit and should on EL8 systems.